CHANGELOG_iOS.md

```diff
@@ -1,5 +1,25 @@
 # Changelog
 
+## [1.77.98](https://github.com/brave/brave-browser/releases/tag/v1.77.98)
+
+ - Implemented an exception list for Night Mode. ([#44213](https://github.com/brave/brave-browser/issues/44213))
+ - Added Yahoo! Japan search engine for new Japan users. ([#43311](https://github.com/brave/brave-browser/issues/43311))
+ - Improved the performance of searching using URL bar for users with many bookmarks, tabs and history items. ([#43703](https://github.com/brave/brave-browser/issues/43703))
+ - Improved search results filtering and auto-complete when using the URL-bar. ([#44842](https://github.com/brave/brave-browser/issues/44842))
+ - Improved cosmetic filtering performance. ([#42530](https://github.com/brave/brave-browser/issues/42530))
+ - Updated Brave Translate to hide an element in reader mode. ([#44835](https://github.com/brave/brave-browser/issues/44835))
+ - Updated Brave Translate onboarding to not include any buttons. ([#44650](https://github.com/brave/brave-browser/issues/44650))
+ - Updated the “Enable Global Privacy Control” settings toggle placement to be at the bottom of the advanced settings page. ([#43690](https://github.com/brave/brave-browser/issues/43690))
+ - Fixed crash when clearing a large amount of history. ([#44981](https://github.com/brave/brave-browser/issues/44981))
+ - Fixed normal tabs being removed when disabling the “Keep Private Tabs” setting toggle. ([#45350](https://github.com/brave/brave-browser/issues/45350))
+ - Fixed an issue where tabs are not blocking pop-ups after switching from another tab. ([#44419](https://github.com/brave/brave-browser/issues/44419))
+ - Fixed empty view when loading Leo Default Model View setting page from the hamburger menu. ([#44072](https://github.com/brave/brave-browser/issues/44072))
```

Comment:
We can probably clean this one up a bit. The above is a bit hard to read. How about something like the following:

```
 - Fixed "Default Model View" page not loading under Brave Leo hamburger settings. ([#44072](https://github.com/brave/brave-browser/issues/44072))
 ```